src/action/grpc.js

```diff
@@ -64,6 +64,17 @@ class GrpcAction {
     this._store.lndReady = true;
   }
 
+  /**
+   * This is called to initialize the GRPC client to autopilot. Once `autopilotReady`
+   * is set to true on the store GRPC calls can be made to the client.
+   * @return {Promise<undefined>}
+   */
+  async initAutopilot() {
+    await this._sendIpc('lndAtplInit', 'lndAtplReady');
+    log.info('GRPC autopilotReady');
+    this._store.autopilotReady = true;
```

Comment:
Didn't see a place where we use `store.autopilotReady`. We can probably remove it from the store until we do.

Reply:
Ah sorry I didn't add this previously but we need it here https://github.com/lightninglabs/lightning-app/pull/1000/files#diff-24b8ff244eb737acf60f151b492a3034R90 to ensure the user doesn't toggle autopilot before the service is ready, and it's also needed for bos scores since otherwise they'll attempt to `SetScores` before the autopilot service is ready.

Reply:
Ah yes good point.